src/csvReaderWrapper.js

```diff
@@ -56,17 +59,20 @@ const isCellValid = (cell) => {
 const updateData = (result, useDefaultColumnDataType, userUploadedData) => {
   var data = result.data;
   var cleanedData = cleanData(data);
+
+  if (cleanedData.length < MIN_CSV_ROWS) {
+    store.dispatch(setInvalidData("tooFewRows"));
+    return;
+  } else if (Object.keys(cleanedData[0]).length < MIN_CSV_COLUMNS) {
+    store.dispatch(setInvalidData("tooFewColumns"));
+    return;
+  }
+
```

Comment:
Just curious, is this a bug fix?

Reply:
Yes, if there were 0 rows or columns, then exceptions would be thrown because `countRemovedRows` has some code which tried to reference the first row.

I think this structure makes more sense anyways because you usually want to check you have valid inputs before running any calculations on them.